src/main/kotlin/org/veupathdb/lib/blast/blastp/BlastP.kt

```diff
@@ -10,168 +10,112 @@ import org.veupathdb.lib.blast.common.fields.*
 interface BlastP : BlastQueryWithIPG {
 
```

Comment:
Stripped out everything but the CLI flag as the actual documentation will now live with the field implementation.